cfe_internal/enterprise/templates/httpd.conf.mustache

```diff
@@ -147,7 +147,11 @@ LogLevel warn
 
   # TLS Compression should be disabled to avoid CRIME
   # https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2012-4929
-  SSLCompression off
+  # SSLCompression off
+  # As part of security hardening we minimize the features provided by OpenSSL.
+  # Case in point, we build openssl 3 without support for compression. As such,
```

Comment:
You might mention the specific configuration option `no-comp` https://wiki.openssl.org/index.php/Compilation_and_Installation just to be super specific. :)